en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,101 @@
+  </div>
+  
+  <div class="rightImageOverview">
+          <a href='{{base_path}}/assets/img/get_started/portals.png'>
+              <img src='{{base_path}}/assets/img/get_started/portals.png' alt="Expose developer-friendly APIs" />
+          </a>
+      </div>
+</div>
 
-#### Developer Portal
 
-![]({{base_path}}/assets/attachments/103327648/126556772.png)
+### Secure your APIs
 
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self-register]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup), discover, evaluate, subscribe to and consume APIs.
+<div style="width: 100%; overflow: hidden;">
 
+<div class="rightContentOverview" style="width:75% !important;">Control your APIs by securing them thoroughly in a user-friendly manner. API Security is more than API authentication and authorization. WSO2 API Manager allows you to secure your APIs fully by using visibility control, threat protection, API payload validation, adhering to well-defined protocols, applying rate limiting policies, and verifying APIs against specifications in addition to API authentication and authorization.
+  </div>
+  
+  <div class="leftImageOverview">
+          <a href='{{base_path}}/assets/img/get_started/security.png'>
+              <img src='{{base_path}}/assets/img/get_started/security.png' alt="Secure your APIs" />
+          </a>
+      </div>
+</div>
 
-The diagram above shows common API consumer lifecycle activities:
 
-#### API Gateway
+### Balance your APIs Demand
 
-The API Gateway is a runtime, backend component (an [API proxy](https://docs.wso2.com/display/EI611/Working+with+APIs) ) developed using [WSO2 ESB]({{base_path}}/getting-started/about-this-release/). API Gateway secures, protects, manages, and scales API calls. It intercepts API requests, applies policies such as throttling and security using handlers, and manages API statistics. Upon validation of a policy, the Gateway passes web service calls to the actual backend. If the service call is a token request, the Gateway passes it directly to the [Key Manager](#key-manager).
+<div style="width: 100%; overflow: hidden;">
 
-The API Gateway supports OAuth 2.0, JWT, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms the above-mentioned authentication mechanisms and enables IT organizations to enforce rate limits and throttling policies.
+<div class="leftContentOverview" style="width:75% !important;">Balancing the load of your system is critical to avoid system outages. WSO2 API Manager provides the capability to add rate limiting policies to your APIs. Furthermore, you can use these policies to monetize your APIs and bring revenue to your organization.  
+  </div>
+  
+  <div class="rightImageOverview">
```

Comment:
Can we get marketing to redraw this image as it shows that it does not follow the same standard as the other images. As the other images have thinner lines.
<img width="856" alt="Screenshot 2020-12-09 at 17 46 48" src="https://user-images.githubusercontent.com/5195851/101628946-96f9b100-3a46-11eb-801f-ddfc396b0a85.png">